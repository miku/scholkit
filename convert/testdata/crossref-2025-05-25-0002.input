{"indexed":{"date-parts":[[2025,5,25]],"date-time":"2025-05-25T00:26:12Z","timestamp":1748132772345},"reference-count":23,"publisher":"BMJ","issue":"6","content-domain":{"domain":["bmj.com"],"crossmark-restriction":true},"short-container-title":["J NeuroIntervent Surg"],"accepted":{"date-parts":[[2018,1,22]]},"published-print":{"date-parts":[[2018,6]]},"DOI":"10.1136\/neurintsurg-2018-013792","type":"journal-article","created":{"date-parts":[[2018,2,12]],"date-time":"2018-02-12T16:09:17Z","timestamp":1518451757000},"page":"507-509","update-policy":"http:\/\/dx.doi.org\/10.1136\/crossmarkpolicy","source":"Crossref","is-referenced-by-count":79,"title":["ELVO: an operational definition"],"prefix":"10.1136","volume":"10","author":[{"given":"Thabele","family":"Leslie-Mazwi","sequence":"first","affiliation":[]},{"given":"Ronil V","family":"Chandra","sequence":"additional","affiliation":[]},{"given":"Blaise W","family":"Baxter","sequence":"additional","affiliation":[]},{"given":"Adam S","family":"Arthur","sequence":"additional","affiliation":[]},{"given":"Muhammad S","family":"Hussain","sequence":"additional","affiliation":[]},{"given":"I Paul","family":"Singh","sequence":"additional","affiliation":[]},{"given":"Don F","family":"Frei","sequence":"additional","affiliation":[]},{"given":"Richard P","family":"Klucznik","sequence":"additional","affiliation":[]},{"given":"Felipe C","family":"Albuquerque","sequence":"additional","affiliation":[]},{"given":"Joshua A","family":"Hirsch","sequence":"additional","affiliation":[]}],"member":"239","published-online":{"date-parts":[[2018,2,8]]},"reference":[{"key":"2024052304241329000_10.6.507.1","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1411587"},{"key":"2024052304241329000_10.6.507.2","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1414792"},{"key":"2024052304241329000_10.6.507.3","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1414905"},{"key":"2024052304241329000_10.6.507.4","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1503780"},{"key":"2024052304241329000_10.6.507.5","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1415061"},{"key":"2024052304241329000_10.6.507.6","doi-asserted-by":"publisher","DOI":"10.1016\/S1474-4422(16)30177-6"},{"key":"2024052304241329000_10.6.507.7","doi-asserted-by":"crossref","unstructured":"Nogueira RG , Jadhav AP , Haussen DC , et al . Thrombectomy 6 to 24 hours after stroke with a mismatch between deficit and infarct. N Engl J Med 2018;378. doi:10.1056\/NEJMoa1706442","DOI":"10.1056\/NEJMoa1706442"},{"key":"2024052304241329000_10.6.507.8","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2015-011707"},{"key":"2024052304241329000_10.6.507.9","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2014-011591"},{"key":"2024052304241329000_10.6.507.10","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2012-010471"},{"key":"2024052304241329000_10.6.507.11","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2016-012515"},{"key":"2024052304241329000_10.6.507.12","doi-asserted-by":"publisher","DOI":"10.1136\/jnis.2009.000166"},{"key":"2024052304241329000_10.6.507.13","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1214300"},{"key":"2024052304241329000_10.6.507.14","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1213701"},{"key":"2024052304241329000_10.6.507.15","doi-asserted-by":"publisher","DOI":"10.1056\/NEJMoa1212793"},{"key":"2024052304241329000_10.6.507.16","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2016-012699"},{"key":"2024052304241329000_10.6.507.17","doi-asserted-by":"publisher","DOI":"10.3174\/ajnr.A4766"},{"key":"2024052304241329000_10.6.507.18","doi-asserted-by":"publisher","DOI":"10.1053\/euhj.2000.2305"},{"key":"2024052304241329000_10.6.507.19","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2015-011952"},{"key":"2024052304241329000_10.6.507.20","doi-asserted-by":"publisher","DOI":"10.1136\/neurintsurg-2016-012275"},{"key":"2024052304241329000_10.6.507.21","doi-asserted-by":"crossref","first-page":"793","DOI":"10.1001\/jamaneurol.2017.0477","article-title":"Association of a primary stroke center protocol for suspected stroke by large-vessel occlusion with efficiency of care and patient outcomes","volume":"74","author":"McTaggart","year":"2017","journal-title":"JAMA Neurol"},{"key":"2024052304241329000_10.6.507.22","doi-asserted-by":"crossref","first-page":"1361","DOI":"10.1001\/jamaneurol.2017.2149","article-title":"Clinical imaging factors associated with infarct progression in patients with ischemic stroke during transfer for mechanical thrombectomy","volume":"74","author":"Boulouis","year":"2017","journal-title":"JAMA Neurol"},{"key":"2024052304241329000_10.6.507.23","doi-asserted-by":"publisher","DOI":"10.1177\/1747493017701147"}],"container-title":["Journal of NeuroInterventional Surgery"],"language":"en","link":[{"URL":"https:\/\/syndication.highwire.org\/content\/doi\/10.1136\/neurintsurg-2018-013792","content-type":"unspecified","content-version":"vor","intended-application":"similarity-checking"}],"deposited":{"date-parts":[[2024,5,23]],"date-time":"2024-05-23T12:03:43Z","timestamp":1716465823000},"score":0.0,"resource":{"primary":{"URL":"https:\/\/jnis.bmj.com\/lookup\/doi\/10.1136\/neurintsurg-2018-013792"}},"issued":{"date-parts":[[2018,2,8]]},"references-count":23,"journal-issue":{"issue":"6","published-online":{"date-parts":[[2018,5,17]]},"published-print":{"date-parts":[[2018,6]]}},"alternative-id":["10.1136\/neurintsurg-2018-013792"],"URL":"https:\/\/doi.org\/10.1136\/neurintsurg-2018-013792","ISSN":["1759-8478","1759-8486"],"issn-type":[{"value":"1759-8478","type":"print"},{"value":"1759-8486","type":"electronic"}],"published":{"date-parts":[[2018,2,8]]}}
