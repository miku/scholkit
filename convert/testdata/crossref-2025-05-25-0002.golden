{
    "source": "crossref",
    "contribs": [
        {
            "extra": {},
            "given_name": "Thabele",
            "raw_name": "Thabele Leslie-Mazwi",
            "role": "author",
            "surname": "Leslie-Mazwi"
        },
        {
            "extra": {},
            "given_name": "Ronil V",
            "raw_name": "Ronil V Chandra",
            "role": "author",
            "surname": "Chandra"
        },
        {
            "extra": {},
            "given_name": "Blaise W",
            "raw_name": "Blaise W Baxter",
            "role": "author",
            "surname": "Baxter"
        },
        {
            "extra": {},
            "given_name": "Adam S",
            "raw_name": "Adam S Arthur",
            "role": "author",
            "surname": "Arthur"
        },
        {
            "extra": {},
            "given_name": "Muhammad S",
            "raw_name": "Muhammad S Hussain",
            "role": "author",
            "surname": "Hussain"
        },
        {
            "extra": {},
            "given_name": "I Paul",
            "raw_name": "I Paul Singh",
            "role": "author",
            "surname": "Singh"
        },
        {
            "extra": {},
            "given_name": "Don F",
            "raw_name": "Don F Frei",
            "role": "author",
            "surname": "Frei"
        },
        {
            "extra": {},
            "given_name": "Richard P",
            "raw_name": "Richard P Klucznik",
            "role": "author",
            "surname": "Klucznik"
        },
        {
            "extra": {},
            "given_name": "Felipe C",
            "raw_name": "Felipe C Albuquerque",
            "role": "author",
            "surname": "Albuquerque"
        },
        {
            "extra": {},
            "given_name": "Joshua A",
            "raw_name": "Joshua A Hirsch",
            "role": "author",
            "surname": "Hirsch"
        }
    ],
    "ext_ids": {
        "doi": "10.1136/neurintsurg-2018-013792"
    },
    "extra": {
        "arxiv": {},
        "dblp": {},
        "openalex": {
            "open_access": {}
        },
        "crossref": {},
        "oai": {}
    },
    "id": "crossref-2609237f013be2120d296a366a9033463d0d5e78",
    "release_date": "50087-10-27",
    "release_type": "article-journal",
    "title": "ELVO: an operational definition"
}